From 151b3c18eb48e8c34478a813dac2bfcc35bd4aa4 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 5 Feb 2022 18:02:53 +0000
Subject: [PATCH] Provide default opkg conf path

If the "-f" option isn't provided by the caller, the current code will
provide an empty "-f" option to opkg, leading to installation failure.

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
 src/Main.cpp.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Main.cpp.in b/src/Main.cpp.in
index d9895b6..5415b3c 100644
--- a/src/Main.cpp.in
+++ b/src/Main.cpp.in
@@ -77,7 +77,7 @@ static std::string s_archIndexFilePath = "/etc/opkg/arch.conf";
 
 static std::string s_command;
 static std::string s_package;
-static std::string s_conf;
+static std::string s_conf = "/etc/opkg/opkg.conf";
 static bool        s_useSysLog = true;
 
 static int 		   s_cleanupCode = 3;			//0 = clean up nothing, 1 = clean up target package , 2 = clean up temp dir   (bitfield)
-- 
2.37.3.windows.1

